### PR TITLE
[CDAP-16959] Avoiding creation of uuid if already present for macros

### PR DESCRIPTION
**Pull request by itsanudeep**
Re-using existing UUID when retrieving macros for display.

JIRA: https://issues.cask.co/browse/CDAP-16959

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Screen Shot 2020-06-15 at 8 51 33 AM
Seems like we removed the delete button from runtime arguments in preview

**Comment by ajainarayanan** [comment]
When there is a unfilled macro during preview, I am unable to stop the preview 
unable_to_stop1

File: cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js [hunk 1]
```diff
@@ -197,8 +197,8 @@ const getRuntimeArgsForDisplay = (currentRuntimeArgs, macrosMap) => {
     currentRuntimeArgs.pairs.forEach((currentPair) => {
       let key = currentPair.key;
       runtimeArgsMap[key] = currentPair.value || '';
-      if (currentPair.notDeletable && currentPair.provided) {
-        providedMacros[key] = currentPair.value;
+      if (currentPair.uniqueId && Object.prototype.hasOwnProperty.call(macrosMap, key)) {
+        macrosWithIds[key] = currentPair.uniqueId;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js]: Can't we combine this and `const macros = ` statement into one? Do we have to reset `currentRuntimeArgs.pairs` 3 times?

**Comment by itsanudeep** [comment]
> Screen Shot 2020-06-15 at 8 51 33 AM
> 
> Seems like we removed the delete button from runtime arguments in preview

The delete button was hidden to the right, had to horizontally scroll. Fixed it.

File: cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js (hunk 1 above)
Comment (itsanudeep) [reply cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js]: Simplified the implementation here. Did not change the implementation in hydrator-service as that needs more work and reusing this implementation won't work there.

**Comment by itsanudeep** [comment]
> When there is a unfilled macro during preview, I am unable to stop the preview
> unable_to_stop1

This behavior exists in develop as well. https://issues.cask.co/browse/CDAP-16972 to fix this.

**Review by ajainarayanan — APPROVED** [review]

**Comment by itsanudeep** [comment]
BUILD: https://builds.cask.co/browse/CDAP-URUT275-1

**Pull request merged**